cuda_core/cuda/core/experimental/_event.py

```diff
@@ -96,6 +114,17 @@ def close(self):
         """Destroy the event."""
         self._mnff.close()
 
+    def __isub__(self, other):
+        return NotImplemented
+
+    def __rsub__(self, other):
+        return NotImplemented
+
+    def __sub__(self, other):
+        # return self - other (in milliseconds)
+        timing = handle_return(driver.cuEventElapsedTime(other.handle, self.handle))
```

Comment:
@carterbox wrote:
> If it's possible to catch and reraise the error with a message like "Timing must be enabled in order subtract Events. Events have timing disabled by default." That would be better than "invalid resource handle".

I believe this could be pretty simple (with less runtime overhead, too):

Roughly, untested:
```
err, timing = driver.cuEventElapsedTime(other.handle, self.handle)
if err == driver.CUresult.CUDA_SUCCESS:
    return timing
if err == driver.CUresult.CUDA_ERROR_INVALID_HANDLE:
    raise RuntimeError("Timing must be enabled in order subtract Events. Events have timing disabled by default.")
handle_return(err)
raise RuntimeError("This code path is meant to be unreachable.")
```

Reply:
updated in commit 268eee5 based on the discussion on Tuesday